tools/project-dashboard/src/client/styles.css

```
@@ -0,0 +1,349 @@
+:root {
+  color: #d8e4e8;
+  background: #070b12;
+  font-family: "IBM Plex Mono", "SFMono-Regular", Consolas, monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)